requirements.txt

```diff
@@ -9,6 +9,7 @@ idna==2.8
 mypy==0.650
 mypy-extensions==0.4.1
 phonenumbers==8.9.16
+Pillow==5.3.0
```

Comment:
This seems not used anywhere

Reply:
This has to be here for ImageField to work: https://stackoverflow.com/questions/25662073/python-django-cannot-use-imagefield-because-pillow-is-not-installed